CHANGELOG.md

```
@@ -1,4 +1,9 @@
-# 7.0.2 (in progress)
+# 7.1.0 (in progress)
+
+##### New Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)